intelmq/bots/outputs/files/output.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python
```

Comment:
I also thought about this, but then decided not to raise this topic because:
They behave quite differently and have 2 sets of parameters. Merging them would mean the user has a longer list of parameters but there are two groups for the two modes which does not make the configuration easier but more complex.